iceberg/bodo_iceberg_connector/iceberg-java/src/main/java/com/bodo/iceberg/BodoIcebergHandler.java

```diff
@@ -166,14 +166,6 @@ public List<Boolean> tableColumnsHaveThetaSketches(String tableId) {
     return hasThetaSketches;
   }
 
-  /**
-   * Updates a Snapshot update operation with the app-id field. This is used to easily identify
-   * tables created by Bodo.
-   */
-  private static void appendAppID(SnapshotUpdate<?> op) {
```

Comment:
Java was complaining about `SnapshotUpdate` for me locally. Removing trivial helpers like this is useful anyways.